core/src/main/java/org/mskcc/cbio/maf/MafUtil.java

```diff
@@ -426,6 +437,14 @@ else if(header.equalsIgnoreCase(T_REF_COUNT) ||
                 normalDepthIndex = i;
             } else if(header.equalsIgnoreCase(NORMAL_VAF)) {
                 normalVafIndex = i;
+            } else if(header.equalsIgnoreCase(DRIVER_FILTER)) {
+            		driverIndex = i;
+            } else if(header.equalsIgnoreCase(DRIVER_FILTER_ANNOTATION)) {
+            		driverAnnIndex = i;
+            } else if(header.equalsIgnoreCase(DRIVER_TIERS_FILTER)) {
+            		driverTiersIndex = i;
+            } else if(header.equalsIgnoreCase(DRIVER_TIERS_FILTER_ANNOTATION)) {
+            		driverTiersAnnIndex = i;
```

Comment:
Whoa .. this monster statement really should be reframed as a hashmap lookup or something. [ in a future cleanup ]